.eslintrc.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
```

Comment:
`.eslintrc` without an extension has been deprecated and should either be .json or .js with the latter being preferred

Reply:
Good one.